client/dashboard/utils/site-plan.ts

```diff
@@ -0,0 +1,89 @@
+import { __ } from '@wordpress/i18n';
+import {
+	chartBar,
+	cloud,
+	megaphone,
+	next,
+	removeBug,
+	search,
+	shield,
+	video,
+} from '@wordpress/icons';
+import { DotcomPlans, JetpackFeatures } from '../data/constants';
+import { hasPlanFeature } from '../utils/site-features';
+import type { Site } from '../data/types';
+
+export const JETPACK_PRODUCTS = [
+	{
+		id: JetpackFeatures.STATS,
+		icon: chartBar,
+		label: __( 'Stats' ),
+		description: __( 'Clear, concise, and actionable analysis of your site performance.' ),
+	},
+	{
+		id: JetpackFeatures.BACKUPS,
+		icon: cloud,
+		label: __( 'VaultPress Backup' ),
+		description: __(
+			'Real-time backups save every change, and one-click restores get you back online quickly.'
+		),
+	},
+	{
+		id: JetpackFeatures.SOCIAL_ENHANCED_PUBLISHING,
+		icon: megaphone,
+		label: __( 'Social' ),
+		description: __(
+			'Auto‑share your posts to social networks and track engagement in one place.'
+		),
+	},
+	{
+		id: JetpackFeatures.CLOUD_CRITICAL_CSS,
+		icon: next,
+		label: __( 'Boost' ),
+		description: __( 'Improves your site speed and performance.' ),
+	},
+	{
+		id: JetpackFeatures.ANTISPAM,
+		icon: removeBug,
+		label: __( 'Akismet Anti-spam' ),
+		description: __( 'Automatically clear spam from comments and forms.' ),
+	},
+	{
+		id: JetpackFeatures.SEARCH,
+		icon: search,
+		label: __( 'Search' ),
+		description: __( 'Instantly deliver the most relevant results to your visitors.' ),
+	},
+	{
+		id: JetpackFeatures.SCAN,
+		icon: shield,
+		label: __( 'Scan' ),
+		description: __( 'Guard against malware and bad actors 24/7.' ),
+	},
+	{
+		id: JetpackFeatures.VIDEOPRESS,
+		icon: video,
+		label: __( 'VideoPress' ),
+		description: __( 'Powerful and flexible video hosting.' ),
+	},
+];
+
+export function getJetpackProductsForSite( site: Site ) {
+	return JETPACK_PRODUCTS.filter( ( product ) =>
+		hasPlanFeature( site, product.id as JetpackFeatures )
+	);
+}
+
+export function getSitePlanDisplayName( site: Site ) {
+	if ( site.plan?.product_slug === DotcomPlans.JETPACK_FREE ) {
+		const products = getJetpackProductsForSite( site );
+		if ( products.length === 1 ) {
+			return products[ 0 ].label;
```

Comment:
Should we prepend `Jetpack ` here?

<img width="300"  alt="Image" src="https://github.com/user-attachments/assets/fd7bae55-976c-4b62-80d2-269cef42e6d2" />

Also because the corresponding page in Jetpack Cloud has `Jetpack Scan` (I wonder how it constructs the name)

<img width="489" height="168" alt="Image" src="https://github.com/user-attachments/assets/ffd77134-5c60-477e-bba1-7189a7bcda0a" />

Reply:
> (I wonder how it constructs the name)

It doesn't, it uses the purchase endpoint where the product_name is Jetpack Scan! I've prepended Jetpack to the individual product names.